openmetadata-ui/src/main/resources/ui/src/components/common/table-data-card-v2/TableDataCardV2.tsx

```diff
@@ -160,7 +161,7 @@ const TableDataCardV2: React.FC<TableDataCardPropsV2> = forwardRef<
               entityType={source.entityType as EntityType}
               icon={serviceIcon}
               openEntityInNewPage={openEntityInNewPage}
-              serviceName={source.serviceType ?? ''}
+              serviceName={(source as Table)?.service?.name ?? ''}
```

Comment:
Why we are type casting the source as Table?

Reply:
Thanks for highlighting this. I was testing this and missed to update main typescript file